frontend/src/components/organisms/MatchesList/MatchesList.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useEffect} from 'react';
+import styled from "styled-components";
+import {Card} from '@material-ui/core';
+import {MatchItem} from "../../molecules/MatchItem/MatchItem";
+import {MIN_CARD_COMPONENT_WIDTH} from "../../atoms/constants/sizes";
+import {MatchListItem} from "./MatchListItem";
+import {MatchesListRestApi} from "../../../restapi/matches-list";
+import {MatchesListDto} from "./MatchesListDto";
+import {MatchStatus} from "./MatchStatus";
+
+const StyledMatchesList = styled(Card)({
+    width: MIN_CARD_COMPONENT_WIDTH,
+});
+
+export type MatchesListProps = {
+    readonly tournamentId: string;
+};
+
+export const MatchesList = ({tournamentId}: MatchesListProps) => {
+
+    const [expanded, setExpanded] = React.useState<string | boolean>(false);
+    const [matchesListItems, setMatchesListItems] = React.useState<MatchListItem[] | undefined>();
+
+    useEffect(() => {
+        MatchesListRestApi()
+            .getMatchesList(tournamentId)
+            .then((matchesListDto) => {
+                const newMatchesListItems = createMatchListItemsFromMatchesListDto(matchesListDto);
+                setMatchesListItems(newMatchesListItems)
+            });
+    }, [tournamentId]);
+
+    const handleChangeExpander = (panel: string | boolean) => (event: any, isExpanded: string | boolean) => {
+        setExpanded(isExpanded ? panel : false);
+    };
+
+    return (
+        <>
+            <StyledMatchesList>
+                {matchesListItems ? matchesListItems.map((item, index) => generateMatchItem(item, index, expanded, handleChangeExpander)) : "Oczekiwanie na pobranie"}
+            </StyledMatchesList>
+        </>
+    )
+};
+
+const generateMatchItem = (matchItem: MatchListItem, index: number, expanded: string | boolean, handleChangeExpander: (panel: string | boolean) => (event: any, isExpanded: string | boolean) => void) => (
+    <MatchItem
+        level={matchItem.level}
+        matchNumber={matchItem.matchNumber}
+        matchStatus={matchItem.matchStatus}
+        onClickTeam={matchItem.onClickTeam}
+        team1={matchItem.team1}
+        team2={matchItem.team2}
+        key={matchItem.matchNumber}
+        expanded={expanded}
+        handleChangeExpander={handleChangeExpander}
+    />
+)
+
+function callBackFunction() {
+    console.log("AaaBbbCcc")
+}
+
+
+
+const createMatchListItemsFromMatchesListDto = (matchesListDto: MatchesListDto): MatchListItem[] => {
+    return matchesListDto.queue.map((matchesItem) => {
+        function findStatus(): string | undefined {
```

Comment:
Moze `statusOf(match: TournamentMatchDto): MatchStatus` A przynajmniej zmiana tego argumentu na koncu, bo w sumie to nawet jest MatchStatus a nie string :)

Reply:
Nie wiem dlaczego, ale ": MatchStatus" wyrzucało mi błąd. Ale sprawdzę jeszcze